src/test/java/uk/gov/digital/ho/hocs/casework/security/AuthorisationAspectTest.java

```diff
@@ -58,167 +58,149 @@ public void setup() {
         MockHttpServletRequest request = new MockHttpServletRequest();
         request.setMethod("GET");
         RequestContextHolder.setRequestAttributes(new ServletRequestAttributes(request));
-        aspect = new AuthorisationAspect(caseService, userService);
+        aspect = new AuthorisationAspect(caseService, caseDataTypeService, userService);
     }
 
     @Test
     public void shouldCaseServiceLookupWhenExistingCase() throws Throwable {
 
-        String type = "MIN";
```

Comment:
The gitblame is me, but I don't like using real data in tests where possible, these tests are not DCU specific behaviour